litmus-portal/frontend/src/pages/HomePage/style.ts

```diff
@@ -67,6 +78,12 @@ const useStyles = makeStyles((theme) => ({
   },
 
   arrowForwardIcon: {
+    color: theme.palette.primary.main,
+    marginLeft: theme.spacing(1),
```

Comment:
Kindly avoid giving margin in negative numbers. Try to rearrange the div or its children position.

Reply:
Here its required @whitetiger1399 , it has been used at many places with negative margins.

Reply:
The optimization may be done in patch releases or else again the changes would increase because of code refactoring.

Reply:
Making this change won't increase the number of file change as it is already being changed. We can take it up as part of refactoring now and not delay it
cc: @arkajyotiMukherjee

Reply:
Delaying it makes no sense... it'll be always put in a backlog... please make all required changes now...

Reply:
Refactoring PRs can be raised during patch releases. cc: @rajdas98

Reply:
ok for the above class i am updating, for all other places we might need another PR @arkajyotiMukherjee

Reply:
It's fine but then you need to take responsibility and track all the files with filenames where the changes need to be done. @ishangupta-ds

Reply:
@whitetiger1399 any reasons for avoiding negative margins? I don't see any harm in using it.